website/pages/docs/from-1.5/functions/contextual/aws_secretsmanager.mdx

```diff
@@ -0,0 +1,56 @@
+---
+layout: docs
+page_title: aws_secretsmanager - Functions - Configuration Language
+sidebar_title: aws_secretsmanager
+description: The aws_secretsmanager function retrieves secrets from Amazon secretsmanager stores.
+---
+
+
+# `aws_secretsmanager_key` Function
+
+Secrets can be read from the [AWS Secrets
+Manager](https://aws.amazon.com/secrets-manager/) and used within your template
+as variables.
```

Comment:
```suggestion
as locals.
```

When using the hcl2_upgrade command to convert the template the function is placed in a variable block which is expected. But was a little confusing to me at first. Your example below is perfect so I would just suggest changing this to `locals`

Reply:
Good call totally, variables cannot be made of functions